draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -29,23 +29,21 @@ normative:
   RFC7519: RFC7519
   RFC8414: RFC8414
   RFC8725: RFC8725
+  RFC9110: RFC9110
 informative:
   RFC6749: RFC6749
-  RFC7521: RFC7521
   ARF:
   	title: "The European Digital Identity Wallet Architecture and Reference Framework"
 
 
 --- abstract
 
-This specification defines a new method of client authentication for OAuth 2.0 {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attestation based authentication scheme.
+This specification defines an extension to the OAuth 2 protocol as defined in {{RFC6749}} which enables a Client Instance to include an attestation in interactions with an Authorization Server or a Resource Server. This new method enables Client Instances involved in a client deployment that is traditionally viewed as a public client, to be able to provide an attestation to an Authorization server or Resource Server in order to authenticate as a Client Instance.
```

Comment:
```suggestion
This specification introduces an extension to the OAuth 2.0 protocol, as outlined in {{RFC6749}}, enabling Client Instances to include an attestation in their interactions with Authorization Servers or Resource Servers. This method allows Client Instances, typically considered as public clients in traditional deployments, to authenticate to an Authorization Server or Resource Server by providing an attestation, thereby enhancing the authentication process for Client Instances.
```

Reply:
I would argue the client instance authenticates as a certain client id from the perspective of the AS. As a test, a client instance could use different JWTs with different confirmation keys but would be still the same client.

Reply:
```suggestion
This specification defines an extension to the OAuth 2 protocol as defined in {{RFC6749}} which enables a Client Instance to include an attestation in interactions with an Authorization Server or a Resource Server. This new method enables Client Instances involved in a client deployment that is traditionally viewed as a public client, to be able to provide an attestation in order to authenticate as an instance of a particular Client.
```